tests/integration/cluster-users/sar-test.sh

```diff
@@ -276,30 +371,192 @@ function runTests() {
 	echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to delete system namespace"
 	testPermissions "delete" "namespace" "${SYSTEM_NAMESPACE}" "no"
 
-	# namespace admin should not be able to create clusterrolebindings - if they can't create it in one namespace, 
+	# namespace admin should not be able to create clusterrolebindings - if they can't create it in one namespace,
 	# that means they can't create it in any namespace (resource is non namespaced and RBAC is permissive)
```

Comment:
now that permissions are tested cluster-wide, the comment is obsolete and should be removed